test/convnets.jl

```diff
@@ -5,178 +5,168 @@ using Flux
 PRETRAINED_MODELS = []
 
 @testset "AlexNet" begin
-  model = AlexNet()
-  @test size(model(rand(Float32, 256, 256, 3, 1))) == (1000, 1)
-  @test_throws ArgumentError AlexNet(pretrain = true)
-  @test_skip gradtest(model, rand(Float32, 256, 256, 3, 1))
+    model = AlexNet()
+    @test size(model(rand(Float32, 256, 256, 3, 1))) == (1000, 1)
+    @test_throws ArgumentError AlexNet(pretrain = true)
+    @test_skip gradtest(model, rand(Float32, 256, 256, 3, 1))
 end
 
 GC.gc()
 
-@testset "VGG" begin
-  @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19], bn in [true, false]
+@testset "VGG" begin @testset "VGG($sz, batchnorm=$bn)" for sz in [11, 13, 16, 19],
+                                                            bn in [true, false]
+
     m = VGG(sz, batchnorm = bn)
 
     @test size(m(rand(Float32, 224, 224, 3, 1))) == (1000, 1)
     if (VGG, sz, bn) in PRETRAINED_MODELS
-      @test (VGG(sz, batchnorm = bn, pretrain = true); true)
+        @test (VGG(sz, batchnorm = bn, pretrain = true); true)
     else
-      @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
+        @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
     end
     @test_skip gradtest(m, rand(Float32, 224, 224, 3, 1))
-  end
-end
+end end
```

Comment:
This looks a little weird, is it a formatter bug?

Reply:
Maybe that's just the formatter's way of saving space 😆 But honestly, I have no idea